testsuite/elytron/src/test/java/org/wildfly/test/integration/elytron/sasl/mgmt/ScramPlusMgmtSaslTestCase.java

```diff
@@ -403,6 +403,10 @@ private static KeyStore loadKeyStore(final File ksFile) throws Exception {
                     .addMechanismConfiguration(plusMechsConfig.withMechanismName("SCRAM-SHA-256-PLUS").build())
                     .addMechanismConfiguration(plusMechsConfig.withMechanismName("SCRAM-SHA-384-PLUS").build())
                     .addMechanismConfiguration(plusMechsConfig.withMechanismName("SCRAM-SHA-512-PLUS").build())
+                    .addMechanismConfiguration(plusMechsConfig.withMechanismName("SCRAM-SHA-1").build())
+                    .addMechanismConfiguration(plusMechsConfig.withMechanismName("SCRAM-SHA-256").build())
+                    .addMechanismConfiguration(plusMechsConfig.withMechanismName("SCRAM-SHA-384").build())
+                    .addMechanismConfiguration(plusMechsConfig.withMechanismName("SCRAM-SHA-512").build())
                     .addMechanismConfiguration(MechanismConfiguration.builder().build()).build());
```

Comment:
The last line here (i.e., ``.addMechanismConfiguration(MechanismConfiguration.builder().build())``) actually ends up adding "undefined" to the mechanism configuration list, which likely wasn't intentional. This line should probably just be removed.